recipes-nodes/rbuilder/rbuilder.inc

```diff
@@ -109,24 +109,24 @@ EXTRA_OECARGO_BUILDARGS += "--features modern --frozen"
 
 do_install:append() {
     install -d ${D}${sysconfdir}/init.d
+    install -d ${D}/persistent/rbuilder
     install -m 0755 ${THISDIR}/init ${D}${sysconfdir}/init.d/rbuilder
 
     # Install config files owned by root, readable by rbuilder
     install -m 0644 ${THISDIR}/config.mustache ${D}${sysconfdir}/rbuilder.config.mustache
     install -m 0640 ${THISDIR}/config.mustache ${D}${sysconfdir}/rbuilder.config
-    install -m 0644 ${THISDIR}/ofac.json ${D}${sysconfdir}/rbuilder.ofac.json
+    install -m 0640 ${THISDIR}/ofac.json ${D}/persistent/rbuilder/rbuilder.blocklist.json
```

Comment:
That will not work at build time because the persistent path will be created at runtime by the disk encryption recipe in meta-confidential-layer.

Reply:
Different approach should be done here to make that file there.
I would recommend to either make it part of an init script that creates that file at runtime after the disk encryption init scripts triggers or make it as part of the config parser prior to parse the secrets

Reply:
why is this line even needed? wouldn't it suffice to manually create an empty file after the encrypted disk was mounted?